tests/installation/grub_test_snapshot.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "y2logsstep";
+use testapi;
+use bmwqemu ();
+
+sub run() {
+    up_and_down();
```

Comment:
oh, already commented the one in before with the same question before seeing this :-)
@MalloZup review http://clean-code-developer.com/Red-Grade.ashx#Dont_Repeat_Yourself_DRY_0 and collect the red wrist band from me tomorrow. I will borrow it to you :-)